### PR TITLE
add support for reading the root password to use from "mkosi.rootpw"

### PR DESCRIPTION
**Pull request by poettering**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -214,6 +214,13 @@ they exist in the local directory:
   script to decide whether to do in in-tree or an out-of-tree build,
   and which build directory to use.
 
+* `mkosi.rootpw` may be file containing the password for the root user
```
Comment (lucaswerkmeister) [inline README.md]: may be **a** file

**Comment by poettering** [comment]
force pushed a new version that fixes both issues. PTAL!

**Comment by lucaswerkmeister** [comment]
Sorry, I found another typo (“pasword”). Force-pushed a fix myself to not further inconvenience you :) if you’re okay with it, I’ll be glad to merge this now.

**Comment by poettering** [comment]
done! thanks a lot for the review!

**Comment by poettering** [comment]
btw, if you are in a reviewing mood, I#d really appreciate a review for #122, too!

**Comment by lucaswerkmeister** [comment]
I’ll try to take a look (tonight or tomorrow), but right now I have to leave, sorry!

**Pull request merged**